experimental/packages/exporter-trace-otlp-grpc/src/OTLPTraceExporter.ts

```diff
@@ -64,4 +58,16 @@ export class OTLPTraceExporter
   getServiceProtoPath(): string {
     return 'opentelemetry/proto/collector/trace/v1/trace_service.proto';
   }
+
+  getUrlFromConfig(config: OTLPGRPCExporterConfigNode): string {
+    if (typeof config.url === 'string') {
+      return config.url;
+    } else if (getEnv().OTEL_EXPORTER_OTLP_TRACES_ENDPOINT.length > 0) {
```

Comment:
These ifs could be collapsed into
```js
return getEnv().OTEL_EXPORTER_OTLP_TRACES_ENDPOINT ||
  getEnv().OTEL_EXPORTER_OTLP_ENDPOINT ||
  DEFAULT_COLLECTOR_URL;
```
Less error-prone since ensures the checked value and returned value are always the same.